src/cmd-sign

```diff
@@ -23,68 +23,81 @@ import boto3
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cosalib.meta import GenericBuildMeta as Meta
 from cosalib.builds import Builds
-from cosalib.cmdlib import (
-    get_basearch,
-    sha256sum_file,
-    import_ostree_commit)
+from cosalib.cmdlib import get_basearch, sha256sum_file, import_ostree_commit
 from cosalib.fedora_messaging_request import send_request_and_wait_for_response
 
-gi.require_version('OSTree', '1.0')
+gi.require_version("OSTree", "1.0")
 from gi.repository import Gio, OSTree
 
 # this is really the worst case scenario, it's usually pretty fast otherwise
 ROBOSIGNATORY_REQUEST_TIMEOUT_SEC = 60 * 60
 
-fedenv = 'prod'
+fedenv = "prod"
 
 
 def main():
     args = parse_args()
     if args.stg:
         global fedenv
-        fedenv = 'stg'
+        fedenv = "stg"
     args.func(args)
 
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--build", help="Build ID", default='latest')
-    subparsers = parser.add_subparsers(dest='cmd', title='subcommands')
+    parser.add_argument("--build", help="Build ID", default="latest")
+    subparsers = parser.add_subparsers(dest="cmd", title="subcommands")
     subparsers.required = True
 
-    robosig = subparsers.add_parser('robosignatory', help='sign with '
-                                    'RoboSignatory via fedora-messaging')
-    robosig.add_argument("--s3", metavar='<BUCKET>[/PREFIX]', required=True,
-                         help="bucket and prefix to S3 builds/ dir")
+    robosig = subparsers.add_parser(
+        "robosignatory", help="sign with " "RoboSignatory via fedora-messaging"
```

Comment:
Looks like it missed merging the two strings here during the line split.

Reply:
will be fixed in the next upload